scripts/build-doc

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+"use strict";
+
+require('./util/work-from-root')();
+
+const docBuilder = require('./util/doc-builder');
+const commandLineOptions = require('commander')
+  .option('-r, --references <items>', 'Comma separated list of references', val => val.split(','))
```

Comment:
@yomexzo what are references and why would I pass them in? This prompt comment might not explain that well enough.